### PR TITLE
feat(cli/installer): Add missing flags for deno install

### LINKED ISSUE #3929: `deno install` does not support import maps
The `deno install` command should support the `--importmap` flag so that you would be able to write:
```
deno install --importmap import_map.json asd asd.ts
```
without it complaining about `Found argument '--importmap' which wasn't expected, or isn't valid in this context`.

### PR DESCRIPTION
**Pull request by nayeemrmn**
Adds:
- `deno install --importmap`
- `deno install --no-remote`
- `deno install --lock`
- `deno install --lock-write`
- `deno install --cached-only`
- `deno install --v8-flags`
- `deno install --seed`
- `deno install --inspect`
- `deno install --inspect-brk`

---

Closes #3929.
Closes #7354.

### CODE REVIEW

**Comment by ry** [comment]
Looks good - thanks @nayeemrmn. I think this should wait for 1.5.0?

**Review by crypt096 — APPROVED** [review]

File: cli/tests/integration_tests.rs [hunk 1]
```diff
@@ -1796,13 +1796,6 @@ itest!(_033_import_map {
   output: "033_import_map.out",
 });
 
-itest!(import_map_no_unstable {
-  args:
-    "run --quiet --reload --importmap=importmaps/import_map.json importmaps/test.ts",
-  output: "import_map_no_unstable.out",
-  exit_code: 70,
-});
```
Comment (bartlomieju) [inline cli/tests/integration_tests.rs:1804]: Why remove this test?

Comment (nayeemrmn) [reply cli/tests/integration_tests.rs:1804]: This is tested in `flags` unit tests now. I also added `.requires("unstable")` on `--import-map` which wasn't there before, I suspect that's why this was mistakenly tested in integration tests.

Comment (bartlomieju) [reply cli/tests/integration_tests.rs:1804]: Nice 👍  thanks

Comment (nayeemrmn) [reply cli/tests/integration_tests.rs:1804]: Oh nevermind I never added a unit test for that flag dependency, but like every other unstable flag I think there's no need for an integration test as long as it's represented in `flags`.

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**